src/mavedb/view_models/score_set.py

```diff
@@ -102,6 +102,14 @@ def target_labels_are_unique(cls, field_value, values):
 
         return field_value
 
+    # Validate that this score set contains at least one target attached to it
+    @validator("target_genes")
+    def at_least_one_target_gene_exists(cls, field_value, values):
+        if len(field_value) < 1:
+            raise ValidationError("Score sets should define at least one target gene.")
```

Comment:
internally, this is represented as target gene, but the error message here should probably correspond to what the user sees in the interface, which is currently 'Target Sequence' and 'Genomic Coordinates'

how about using 'target' instead of 'target gene' here?